numba/tests/test_ir.py

```diff
@@ -432,7 +432,7 @@ def check_diffstr(string, pointing_at=[]):
             blk = z_ir.blocks[label]
             ref = blk.body[:-1]
             idx = None
-            for i in range(len(ref)):
+            for i in range(len(ref) - 1):
```

Comment:
Is this just error in test logic?

Reply:
Yes. It's supposed to be looking for two adjacent `Del` nodes, the loop body contains an index into `ref` at `i + 1` so as to look ahead one index, hence the `range` needs limiting so as to not lookahead out of bounds.